lib/plan-builder-generated.js

```diff
@@ -6942,6 +6990,154 @@ bindParam(...args) {
 
     }
 }
+/**
+  * PatchBuilder objects have methods.
+  * @namespace planBuilder.PatchBuilder
+  * @since 3.4.0
+  */
+class PlanPatchBuilder extends types.ServerType {
+
+  constructor(ns, fn, args) {
```

Comment:
I think this needs the same 3 constructors that `PatchBuilderImpl` has:

1. One that accepts just a context path.
2. One that accepts a context path and a map of namespaces.
3. A private one that accepts a list of args. 

Each constructor should then always build a new "op/suboperators" object and add args to it.